src/Codeception/Module/WebDriver.php

```diff
@@ -404,10 +408,8 @@ class WebDriver extends CodeceptionModule implements
     protected $test;
     protected $sessions = [];
     protected $sessionSnapshots = [];
-    protected $httpProxy;
```

Comment:
These properties were created at the following commit.

https://git.hrzg.de/t.munk/Codeception/commit/43f6e26d5f010dd3aa390252e3d7a10efd733c48?w=1

However, it's not used from anywhere.

Current `httpProxy` and `httpProxyPort` option are passed to capabilities (to put an proxy between webdriver server and SUT).

Reply:
sslProxy and sslProxyPort is also.

Reply:
Good point, it seems that these properties have never been used.